.circleci/config.yml

```diff
@@ -17,3 +17,4 @@ jobs:
       - run: sbt +compile
       - run: sbt +test:compile
       - run: sbt +test
+      - run: sbt "dumpEventSchemaCompilerPlugin / publishLocal" scripted
```

Comment:
Ermmm... that still doesn't clarify the purpose of this step to the future non-sbt-expert reader 🤔